### PR TITLE
Rolling mode offset

### LINKED ISSUE #2948: [Feature Request] Allow rolling mode to take percentage
Instead of having rolling mode always be in the centre of timeline add an option to take percentage from left side.

### PR DESCRIPTION
**Pull request by Smileydude**
Allow rolling mode to take in a percentage from left as an option

fixes #2948

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,11 +0,0 @@
-**Thank you for contributing to vis.js!!**
```
Comment (bradh) [inline .github/PULL_REQUEST_TEMPLATE.md]: Don't think this file should be deleted.

**Comment by mojoaxel** [comment]
Aggrrr...This merged the master into develop, what we don't do. We have to revert this. I'll cleanup the develop branch.

@bradh Please wait at least one more review before merging!

**Comment by mojoaxel** [comment]
@bradh Sorry every thinks looks good. My mistake.

**Comment by Smileydude** [comment]
Sorry ya some commit history would be merged over. I didn't notice that this project was not developed in master till I found the checklist. I did clear out any changes that would have came from master. 
My next pull request will be developed in the right branch :)

**Pull request merged**